ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/eclipse/persistence/main/module.xml

```diff
@@ -2,7 +2,7 @@
 
 <!--
   ~ JBoss, Home of Professional Open Source.
-  ~ Copyright 2011, Red Hat, Inc., and individual contributors
+  ~ Copyright 2021, Red Hat, Inc., and individual contributors
```

Comment:
It should not block the merge but FYI we don't proactively bump to the current year,

Reply:
@darranl thanks for your feedback. I'm going to avoid these kind of change in the future.